cupy/linalg/product.py

```diff
@@ -7,6 +7,7 @@
 
 from cupy.linalg.solve import inv
 from cupy.util import collections_abc
+from cupy.util import _normalize_axis_index
```

Comment:
Please do not import a function, as per [the OpenStack style guideline](https://docs.openstack.org/hacking/latest/user/hacking.html#imports).

```suggestion
from cupy import util
```

Reply:
@niboshi Yup fixed it